### ISSUE TITLE
Performance issue using canvas.toDataURL?

### ISSUE THREAD

**Issue by bnolan** [maxkrieger/aframe-draw-component#3]
Would it be faster to set the texture directly from the canvas by manipulating the Object3D.material?

**Comment by maxkrieger** [comment]
Is this possible? I'm not familiar with threejs.

**Comment by bnolan** [comment]
Yup, definitely possible, something like:

```
    var texture1 = new THREE.Texture(canvas1) 
    texture1.needsUpdate = true;
```

This is an example of setting a texture from a canvas.

https://stemkoski.github.io/Three.js/Texture-From-Canvas.html

Great component by the way, I've been using it for my [htmltexture component](https://scenevr.github.io/htmltexture-component/basic/index.html).

**Comment by maxkrieger** [comment]
ah, I'll implement that tomorrow. Would probably solve https://github.com/maxkrieger/aframe-draw-component/issues/2 too.

I saw htmltexture, and featured it in the readme! Exciting to see us nearing "endgame" already.

**Comment by maxkrieger** [comment]
Just released `1.3.0` which implements this! Thx!
